src/assets/styles/_width-height.scss

```diff
@@ -1,4 +1,50 @@
-.fit              { max-width: 100%; }
-.half-width       { width: 50% }
-.full-width       { width: 100%; }
-.full-height      { height: 100%; }
\ No newline at end of file
+.fit { 
+  max-width: 100%;
+}
+
+.half-width { 
+  width: 50%; 
+}
+
+.full-width { 
+  width: 100%; 
+}
+
+.full-height { 
+  height: 100%; 
+}
+
+html,
+body {
+  width: 100%;
+  height: 100%;
+  -webkit-tap-highlight-color: transparent;
```

Comment:
Move this to some other file, as it's not related to width or height.

Reply:
@abhaychawla I have moved this to miscellaneous file.